recipes/main/schnitzel_viennese_style.adoc

```diff
@@ -0,0 +1,48 @@
+[id='sec.schnitzel_viennese_style']
+
+
+ifdef::env-github[]
+:imagesdir: ../../images
+endif::[]
+ifndef::env-github[]
+:imagesdir: images
+endif::[]
+
+=== Schnitzel Viennese Style
+
+image::schnitzel_viennese_style/schnitzel.jpg[Name, 640, 480]
+
+Contributed by: Robert
+image:contributors/robert_s.png[Name, 140, 140]
+
+Schnitzel is very common in Germany, the original however is from Austria. Schnitzel Viennese (Wiener Schnitzel) has to be made of veal, the more popular made of pork (rarely turkey) is called Viennese Style (Wiener Art). Making is very much the same and since it's crumbed, there's usually no sauce with it.
```

Comment:
```suggestion
Schnitzel is very common in Germany, the original however is from Austria. Authentic Schnitzel Viennese (Wiener Schnitzel) has to be made from veal, the more popular variation made from pork (rarely turkey) is called Viennese Style (Wiener Art). Making is very much the same and since it's coated in breadcrumbs, there's usually no sauce with it.
```

Reply:
thx for the suggestions, mostly adjusted...